pkg/volume/csi/csi_plugin.go

```diff
@@ -625,6 +709,26 @@ func (p *csiPlugin) skipAttach(driver string) (bool, error) {
 	return false, nil
 }
 
+// getDriverMode returns the driver mode for the specified spec: {persistent|ephemeral}.
+// 1) If mode cannot be determined, it will default to "persistent".
```

Comment:
this statement is not true. If mode cannot be determined (nothing in the spec) the code throws an error. Please update comment or code